Automattic/jetpack
# Add site lifecycle status guess to My Jetpack

## Proposed changes:
- Add all stats needed to perform user/site lifecycle status check
- Add selector combining all available information and doing best guess 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
* Go to /wp-admin/admin.php?page=my-jetpack
* Check that `window.myJetpackInitialState.userStats` includes correct information
* Do a static code analysis of selector `getGuessedSiteLifecycleStatus()` whether it matches the following conditions:
```
"Brand New Sites"
Have no paid products
Have no active Jetpack modules (aside from modules that are active by default)
Have only one Jetpack plugin installed
Have no site or user connection
May or may not come from a known host (we can do our best to guess)

"New Sites"
Have no paid products
Have 3 or fewer Jetpack modules activated

"Settling-in Sites"
Have one paid product (this does not include bundles - which contain multiple products)
Have fewer than 10 active Jetpack features (paid or free)

"Established Sites"
Have more than one paid product (could be individual products or a bundle) OR
Have 10+ Jetpack features enabled
```
